libs/template/file.go

```diff
@@ -53,6 +57,10 @@ type copyFile struct {
 	srcPath string
 }
 
+func (f *copyFile) isNotebook() bool {
```

Comment:
`notebook.Detect` won't work because in the `MemoryFile` version we don't have a file on disk. We could re-write the detection code to note rely on `fs`.

Reply:
Updated `notebook.Detect` to work with in-memory files